proto/topo.proto

```diff
@@ -81,6 +81,34 @@ message Node {
   // If interfaces is empty the interfaces defined in the links portion of the
   // topology will be populated into the node.
   map<string, Interface> interfaces = 12;
+  // Any host vm constraints required by the vendor.
+  repeated HostConstraint host_constraints = 13;
+}
+
+// HostConstraint includes all the necessary system constraints required 
```

Comment:
HostConstraint is a constraint on the host where the topology is running.

Reply:
Fixed.